packages/performance/src/services/perf_logger.test.ts

```diff
@@ -17,7 +17,7 @@
 
 import { stub, SinonStub, useFakeTimers, SinonFakeTimers } from 'sinon';
 import { Trace } from '../resources/trace';
-import * as ccService from './cc_service';
+import * as ccService from './transport_service';
```

Comment:
Since we are replacing all the context of CC to Transport, should we import it as transportService instead of ccService?

Reply:
Updated, thank you!